src/static/js/modules/nav-primary.js

```diff
@@ -17,7 +17,16 @@ function init() {
 
   /* TODO: Add Unit Tests */
   $primaryTrigger.on( 'click', function() {
-    es.set.toggleExpandedState( $primaryNav );
+    if ( es.get.isThisExpanded( $primaryNav ) ) {
+      es.set.toggleExpandedState( $primaryNav, 'false', function() {
+        $primaryTrigger.focus();
+      } );
+    } else {
+      es.set.toggleExpandedState( $primaryNav, 'true', function() {
+        $primaryLink[0].focus();
```

Comment:
Oh, sorry, because there are multiple `$primaryLink`s but only one `$primaryTrigger` (L:22 is a diff object) and we want to ensure it always focuses the first link in the list (once we get rid of the TBD non-sense).